components/ticktick/actions/update-task/update-task.mjs

```diff
@@ -0,0 +1,67 @@
+import ticktick from "../../ticktick.app.mjs";
+import { removeNullEntries } from "../../common/utils.mjs";
+
+export default {
+  key: "ticktick-update-task",
+  name: "Update a Task",
+  description: "Update a Task.[See doc](https://developer.ticktick.com/api#/openapi?id=update-a-task)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    ticktick,
+    taskId: {
+      type: "string",
+      label: "Task ID",
+      description: "ID of task to update",
+    },
+    title: {
+      type: "string",
+      label: "Title",
+      description: "Task title",
+    },
+    content: {
+      type: "string",
+      label: "Content",
+      description: "Task content",
+      optional: true,
+    },
+    allDay: {
+      type: "boolean",
+      label: "All day",
+      description: "All day",
+      reloadProps: true,
+      optional: true,
+    },
+  },
+  async additionalProps() {
+    const props = {};
+    props.startDate = {
+      type: "string",
+      label: "Start date",
+      description: "Start date and time in \"yyyy-MM-dd'T'HH:mm:ssZ\" format. Example : \"2019-11-13T03:00:00+0000\"",
+      optional: true,
+    };
+    props.dueDate = {
+      type: "string",
+      label: "Due date",
+      description: "Due date and time in \"yyyy-MM-dd'T'HH:mm:ssZ\" format. Example : \"2019-11-13T03:00:00+0000\"",
+      optional: true,
+    };
+    return props;
+  },
```

Comment:
Do these need to be `additionalProps`? Couldn't they be listed with the main props?

Reply:
I don't think they should be listed. I am meant to include a control there